Pin the address of the Y-build repo to 4.35-Y-builds

stephan-herrmann opened the pull request:
Related to
https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/2927

stephan-herrmann commented:
Something has gone wrong with the baseline for branch builds, see also https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/2089

As we don't control the baseline I'll blindly apply all version bumps requested by tycho.

stephan-herrmann commented:
note to self: when / if merging correct the commit message (4.34 -> 4.35).

stephan-herrmann commented:
Given we have a successful Y-build (so it seems), this PR can be abandoned, right @jarthana ? Or should we wait for [tests](https://ci.eclipse.org/releng/job/YPBuilds/job/ep435Y-unit-linux-x86_64-java24/74/) to complete?

(same then for jdt.ui and jdt.debug).

jarthana commented:
> Given we have a successful Y-build (so it seems), this PR can be abandoned, right @jarthana ? Or should we wait for [tests](https://ci.eclipse.org/releng/job/YPBuilds/job/ep435Y-unit-linux-x86_64-java24/74/) to complete?
> 
> (same then for jdt.ui and jdt.debug).

I was wondering the same. Let me do that. Tests shouldn't matter in this case, right?

stephan-herrmann commented:
> > Given we have a successful Y-build (so it seems), this PR can be abandoned, right @jarthana ? Or should we wait for [tests](https://ci.eclipse.org/releng/job/YPBuilds/job/ep435Y-unit-linux-x86_64-java24/74/) to complete?
> > (same then for jdt.ui and jdt.debug).
> 
> I was wondering the same. Let me do that. Tests shouldn't matter in this case, right?

In the worst case, tests would reveal a stop-ship bug, in which case it would be _convenient_ to push a hotfix via PR with verification by a PR build. Do we trust the fact that we made no code changes since last successful tests, and hence tests "shouldn't" fail? :)

stephan-herrmann commented:
With [tests passing on two platforms](https://github.com/eclipse-jdt/eclipse.jdt.core/issues/2899#issuecomment-2734404255) we don't seem to need this PR anymore.
